apps/console/src/app/lib/theme/ant-theme.ts

```diff
@@ -21,7 +22,7 @@ export const antTheme: ThemeConfig = {
       fontFamily: "Brockmann",
     },
     Layout: {
-      colorBgHeader: "#141414",
-    },
+      colorBgHeader: "#141414"
```

Comment:
Try to see if there's a tailwind color that looks identical/similar, we use tailwind colors in multiple places